Core1Platform/AdministratingCore1/06_ExpertKnowledge.md

```diff
@@ -0,0 +1,106 @@
+# Expert knowledge
+
+This section contains procedures that you only should use, if you really know the consequences of your actions well.
+
+
+
+## Switch on &frasl; off single feature flags
+
+The Entwicklermodus (Development mode) of the engine room allows to switch on &frasl; off specific system behavior fundamentally.   
+For example, you can enable special caching functions of the *Core1 Platform*. Additionally, you can disable module-specific features such as the import of orders and event manager features.   
+You can use the development mode for testing purposes or, for example, to temporarily switch off a connection, because you are informed about a third-party application downtime. Each action in the development mode is logged.
+
+> [WARNING] Switching off module features can result in data not being transferred to third party applications, and in this case it can lead to data loss. If you are not really sure about the consequences of switching off a feature, you should not use it. 
+
+The development mode contains core features as well as engine-, module-, and driver-specific feature flags. For example, if you switch off the workflow, the following will happen:
+- The workflow is stopped.
+- All current processes do no longer import and 
+output data. 
+<!---Julian, gibt es noch ein anderes, drastisches Beispiel?-->
+
+#### Prerequisites
+
+- You have administrator rights for the current instance.
+- You know the consequences of switching on &frasl; off a feature.
+
+#### Procedure
+
+*Actindo Core1 Platform > Any workspace*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened. The current workspace is overlapped.
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the *Entwicklermodus (Development mode)* drop down top right.   
+   The list of feature flags is expanded. Each feature flag is listed in a feature chip with a toggle. The list of feature chips with toggles contains general feature flags of the *Core1 Platform* as well as engine-, module-, and driver-specific feature flags.
```

Comment:
Each feature flag is listed in a feature chip with a toggle. --> feature flag vs. feature chip
Was ist mit feature chip gemeint? Was ist der Unterschied zwischen den beiden? Was ich sehe neben den "Features" sind nur Toggles.